### ISSUE TITLE
Cannot run with gradle:4.1.0

### ISSUE THREAD

**Issue by Hi99simon** [mtellect/CameraDeepAR#46]
Switch from 
classpath 'com.android.tools.build:gradle:3.5.0'
to
classpath 'com.android.tools.build:gradle:4.1.0'

It caused "Direct local .aar file dependencies are not supported when building an AAR" error.

Any updates?

**Comment by mtellect** [comment]
Drop a screenshot of your project build.gradle && app build.gradle

**Comment by mtellect** [comment]
And please note , if you follow the example on the Dev branch exactly as it is, you won't have issues.
This plugin was built for a particular purpose in mind and a specific feature on Deepar SDK (hair segmentation).

**Comment by mtellect** [comment]
Drop a screenshot of your gradle-wrapper.properties

**Comment by Hi99simon** [comment]
Yes, I use the same example project, and upgrade from 3.5.0 to 4.1.0 gradle version, since other plugins require higher gradle version.

And if you run it, it says that "Direct local .aar file dependencies are not supported when building an AAR." 

Do you have any solutions?

THANKS SO MUCH!

<img width="1425" alt="Screenshot 2022-01-29 at 9 58 40 AM" src="https://user-images.githubusercontent.com/41977267/151642945-332d3915-9756-4113-a10c-059f3d9156b4.png">
<img width="1432" alt="Screenshot 2022-01-29 at 9 58 55 AM" src="https://user-images.githubusercontent.com/41977267/151642949-6728eb9b-6064-4cb0-9a6f-22be67b0c93e.png">

**Comment by Hi99simon** [comment]
And after running, errors like this shown. Can you replicate this error?
<img width="1432" alt="Screenshot 2022-01-29 at 9 58 55 AM" src="https://user-images.githubusercontent.com/41977267/151643509-4314d052-b7b6-4bcb-bb75-13caded1a35f.png">
.

**Comment by mtellect** [comment]
Hi, if you do want to use this plugin set the following:

Your project build.gradle classpath : 

classpath 'com.android.tools.build:gradle:3.5.0'

set your app.gradle to:

compileSdkVersion 29
minSdkVersion 21
targetSdkVersion 29

set your gradle/wrapper/gradle-wrapper.properties to:
distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
